src/components/Accordion/Accordion.tsx

```diff
@@ -3,11 +3,14 @@ import React, { useId } from 'react';
 import type { ClickHandler } from './Context';
 import { AccordionIconVariant, AccordionContext } from './Context';
 import classes from './Accordion.module.css';
+import type { AccordionHeaderProps } from './AccordionHeader';
+import AccordionHeader from './AccordionHeader';
```

Comment:
We should prefer using named imports/exports instead of default exports. There are some reasons [why default exports should be avoided here](https://basarat.gitbook.io/typescript/main-1/defaultisbad) (some reasons are better than others, f.ex typo protection and auto imports).

I see that `AccordionHeader` actually has a default export, which it shouldnt have. Would be great if you could also remove that 🙂

Ill add a task to get some eslint rules set up to avoid this in the future

Reply:
Done 🙂